server/resources/policy_filter_test.go

```diff
@@ -0,0 +1,266 @@
+package resources
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Policy filter", func() {
+	DescribeTable("Apply filter to resource",
```

Comment:
I like those test cases. But 2 largest ones barely fit on a screen. Can we compact them a bit?

For example:
```
{
    "property": "a",
    "type":     "eq",
    "value":    1,
}
```
could become:
```
equals("a", 1)
```